### PR TITLE
Improving GitHub Action + README

### PR DESCRIPTION
**Pull request by wesleyhf**
## Github Actions

- docker buildx added
- cache added
- builds in parallel

## README

- topics re-ordered
- all available tags (with link to Dockerfile)
- all environment variables
- badge added
- fixing `adonis` typo

## ~Important~

~I removed the latest tag on github actions, should I put it back?~

### CODE REVIEW

**Review by fabriciojs — COMMENTED** [review]
About the `latest` tag - while I am fan of not having it possibly, since we already have, in order to stop building it, we need to think - either we continue building the latest tag, or we have to remove the latest (since we are not gonna build it anymore, the current latest would not, you know, the latest)... in such case - removing the latest - we could break somewhere that is currently relying on it (need to check kool presets for using latest node or something).

What you think @dbpolito ? I guess the easiest is to keep building the `latest`.

**Comment by dbpolito** [comment]
I'm not a huge fan of latest too but i see it's value, like running with no tag at all... let's keep it... we won't use it but might be useful for others

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (dbpolito) [inline .github/dependabot.yml:12]: no new line

File: README.md [hunk 2]
```diff
@@ -1,50 +1,77 @@
...
+- [14](https://github.com/kool-dev/docker-node/blob/master/14/Dockerfile)
+- [14-qa](https://github.com/kool-dev/docker-node/blob/master/14-qa/Dockerfile)
+- [14-adonis](https://github.com/kool-dev/docker-node/blob/master/14-adonis/Dockerfile)
+- [14-adonis-qa](https://github.com/kool-dev/docker-node/blob/master/14-adonis-qa/Dockerfile)
+- [14-nest](https://github.com/kool-dev/docker-node/blob/master/14-nest/Dockerfile)
+- [14-nest-qa](https://github.com/kool-dev/docker-node/blob/master/14-nest-qa/Dockerfile)
+
+## Environment Variables
+
+Variable | Description | Default Value
```
Comment (dbpolito) [inline README.md:38]: change order

**Comment by wesleyhf** [comment]
@dbpolito @fabriciojs I added the latest tag back. Could you guys review it again?

**Review by fabriciojs — APPROVED** [review]

**Pull request merged**